docs/swiper.md

```diff
@@ -0,0 +1,500 @@
+---
+id: swiper
+title: Swiper
+---
+
+### Usage
+
+```js
+import { Swiper } from 'react-native-elements'
+
+<Swiper>
+  <View style={{ flex: 1 }}>{/* any content */}</View>
+  <View style={{ flex: 1 }}>{/* any content */}</View>
+  {/* ... */}
+</Swiper>
+
+<Swiper
+  vertical {/* slide up / down instead left / right */}
+  from={1} {/* initial slide is second */}
+  loop {/* back to first slide after last */}
+  timeout={2} {/* autoplay duration (2 secs) */}
+  springConfig={{ speed: 11 }} {/* RN Animated.spring config */}
+  minDistanceForAction={0.15} {/* Swipe less that 15% keep active slide */}
+  positionFixed {/* Fix mobile safari vertical bounces */}
+>
+  <View style={{ flex: 1 }} /> {/* Slide 1 */}
+  <View style={{ flex: 1 }} /> {/* Slide 2 */}
+  {/* ... */}
+</Swiper>
+```
+
+---
+
+## Props
+
+- [`vertical`](#vertical)
+- [`from`](#from)
+- [`loop`](#loop)
+- [`timeout`](#timeout)
+- [`gesturesEnabled`](#gesturesenabled)
+- [`springConfig`](#springconfig)
+- [`minDistanceToCapture`](#mindistancetocapture)
+- [`minDistanceForAction`](#mindistanceforaction)
+- [`positionFixed`](#positionfixed)
+- [`containerStyle`](#containerstyle)
+- [`innerContainerStyle`](#innercontainerstyle)
+- [`swipeAreaStyle`](#swipeareastyle)
```

Comment:
Is there any way possible that this could be simplified? 3 layers of wrappers is a lot

containerStyle -> innerContainerStyle -> swipeAreaStyle

Based on previous experience, users always get confused when they're multiple wrappers.

Reply:
Because of web. CSS's `position` is not equal to RN `position`. Besides this, we cannot know the styles of the root container. Eg [expo makes `#root` rubber](https://github.com/expo/expo/issues/5081) by default.

I have been looking for a solution for a long time but cannot exclude innerContainer.